source/SH3/system/glprogram.cpp

```diff
@@ -55,6 +55,50 @@ GLint sh3_glprogram::Load(const std::string& name)
     glDeleteShader(fragShader);
 }
 
+GLint sh3_glprogram::Load(const std::string& name, std::vector<std::string> const &attribs)
```

Comment:
Done. I forgot you can do this, I'm still getting used to some more modern c++ stuff sorry.